src/node-lib/curriculum-api-2023/queries/pupilLesson/pupilLesson.gql

```diff
@@ -39,5 +39,6 @@ query pupilLesson(
     exit_quiz_id
     supervision_level
     video_title
+    has_worksheet_asset_object
```

Comment:
seemed silly that we were doing two api calls when we had this data already so I've added it to the query